language/Portuguese_Brazil/strings.po

```diff
@@ -516,27 +516,27 @@ msgstr "Opções do State/Playback"
 
 #: src/frontend/mame/ui/submenu.cpp:76
 msgid "Automatic save/restore"
-msgstr "Salve/restaure automático"
+msgstr "Salvamento/Restauração Automáticos"
```

Comment:
"Salvamento" is a very ugly word :-P

Maybe here it could be better to say "Salvar e restaurar automaticamente"